packages/subgraph/src/kpi-tokens-manager.ts

```diff
@@ -76,6 +84,37 @@ export function getTemplate(
     return KPITokenTemplate.load(templateId(id, version));
 }
 
+function getTemplateFeature(
+    templateId: BigInt,
+    onChainFeatureId: BigInt,
+): KPITokenTemplateFeature {
+    const id = featureId(
+        getKPITokensManager().id,
+        templateId,
+        onChainFeatureId,
+    );
+    let feature = KPITokenTemplateFeature.load(id);
+    if (feature == null) {
+        feature = new KPITokenTemplateFeature(id);
+        feature.featureId = onChainFeatureId;
+        feature.paused = false;
+
+        const oracleManager = getKPITokensManager();
```

Comment:
`const oracleManager` should be `const kpiTokenManager`, no? Since we are using `getKPITokensManager()`.

Reply:
Oh yes, absolutely. Fixed in 0dedba60ec462b3ca183a04719fe5d88346aa841